libs/blocks/locui/utils/miloc.js

```diff
@@ -136,11 +136,18 @@ export async function cancelProject() {
   return resp.status;
 }
 
-export async function rolloutLang(languageCode, reroll = false) {
```

Comment:
Nit: Is there a reason for changing the function name? I feel like `rolloutLang` sounds fine.

Reply:
I guess I was just anticipating some disagreement over the semantics of it, since (as far as I can tell) Retrying does not necessarily mean rolling out 🤷‍♂️ I don't have a problem changing it back though if I'm wrong about that

Reply:
What does retry (on the lang card) do other than re-attempt rollout?

Reply:
This would be a question for @jenssingler. My original version of this button did just that, but apparently we need this to go to a special `retry` endpoint for other purposes.

Reply:
Also worth mentioning -- I have absolutely no problem making this a separate function and leaving `rolloutLang` alone 😂 it'll just be repeating some code, which I was hoping to avoid. But if we're all in agreement that that's okay, I'm happy to change it back.